Python/import.c

```diff
@@ -2230,8 +2230,14 @@ PyImport_LoadLazyImport(PyObject *lazy_import)  // was PyImport_ImportDeferred(P
     PyObject *obj = lz->lz_obj;
     if (obj == NULL) {
         obj = _imp_load_lazy_import_impl(lz);
-        if (obj != NULL)
+        if (obj != NULL) {
             lz->lz_obj = obj;
+        }
+        else {
+            _PyErr_FormatFromCause(PyExc_LazyImportError,
+                                   "Improper Module import causes this error "
+                                   "when enabling Lazy Imports.");
```

Comment:
change error message to something like "Error occurred when loading a lazy import. Original import was at {file} {line}" (which would require adding file and line metadata to the lazy import object)